pulsar-broker/src/main/java/org/apache/pulsar/broker/admin/AdminResource.java

```diff
@@ -321,6 +321,9 @@ protected void validateGlobalNamespaceOwnership(String property, String namespac
         } catch (IllegalArgumentException e) {
             throw new RestException(Status.PRECONDITION_FAILED, "Tenant name or namespace is not valid");
         } catch (RestException re) {
+            if (re.getResponse().getStatus() == Status.NOT_FOUND.getStatusCode()) {
```

Comment:
@jiazhai @aloyszhang

thanks for providing a fix to my issue.
 
I am not sure I understand why we are converting all of the RestExceptions to "Namespace does not have any clusters configured"
what about 5xx errors ("Internal server error") ?

probably we should forward to PRECONDITION_FAILED only under specific circumstances.